src/Form/CommitteeCommandType.php

```diff
@@ -26,7 +26,6 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('description', TextareaType::class, [
                 'filter_emojis' => true,
-                'purify_html' => true,
```

Comment:
why do you remove this? `purify_html` filter the content of the field and keep whitelisted html tags.

Reply:
Because the content is display without raw and it cause some problems with aposthrophe

Reply:
maybe you can just change the flag [here](https://github.com/EnMarche/en-marche.fr/blob/develop/src/Utils/HtmlPurifier.php#L25)
what do others think?

Reply:
@illusionOfParadise Don't we had the same sort of issue somewhere else in the past ?

Reply:
Yes, I did the same thing for citizen project.

Reply:
@ottaviano change the flag can maybe impact too many form no ? :s